controllers/cluster_controller.go

```diff
@@ -747,3 +756,32 @@ func (r *FoundationDBClusterReconciler) getCoordinatorSet(cluster *fdbv1beta2.Fo
 
 	return adminClient.GetCoordinatorSet()
 }
+
+// restartFdbserverProcess will restart the fdbserver process for the provided Pod.
+func (r *FoundationDBClusterReconciler) restartFdbserverProcess(name string, namespace string) error {
+	execReq := r.RestClient.
+		Post().
+		Namespace(namespace).
+		Resource("pods").
+		Name(name).
+		SubResource("exec").
+		VersionedParams(&corev1.PodExecOptions{
+			Container: "foundationdb",
+			Command:   []string{"sh", "-c", "pkill fdbserver"},
```

Comment:
Validate that we are only killing the fdbserver if it points to the default version.